### ISSUE TITLE
Ability to edit doc HTML

### ISSUE THREAD

**Issue by dgarijo** [OnToology/OnToology#535]
This feature is 2 sub features:
- Be able to edit the WIDOCO metadata config file from OnToology, including hitting "create PR" or "commit changes" from the editor and creating a new pull request in the repo.
- Be able to edit the generated WIDOCO html and previsualize it.

Why? Right now the workflow is: 1) create onto, 2) upload to github, 3) Wait for OnToology  to generate PR; 4) check doc online, 5) Clone the repo, fix doc manually. It is easier if I can previsualize the generated doc in OnToology and edit changes in case of need. Changes would be a new PR.

This requires a new view with a **simple** HTML form.

**Comment by ahmad88me** [comment]
Ya, maybe we can split it later into two different issues. and I think the previsual one is going to be really helpful.

**Comment by dgarijo** [comment]
Yes, I think you don't need to do everything @ahmad88me. We could divide this a little bit so other students/programmers would contribute. We should meet at some point to discuss this and other ideas I have

**Comment by ahmad88me** [comment]
Yes, it would be much help to have more contributors. We can talk about it whenever you have time.
